Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-listIndexes.txt

MongoDB\\Collection::listIndexes()
==================================

.. meta::

>    :description: Retrieve information on all indexes for a collection using the `listIndexes()` method in PHP.

Reviewer: ```suggestion
   :description: Retrieve information about all indexes in a collection with the MongoDB PHP Library.
```
